projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-input/search-ai-input.component.scss

```diff
@@ -71,6 +71,13 @@ aca-search-ai-input {
     &-content {
       display: flex;
       align-items: baseline;
+
+      &-text {
+        width: 120px;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
Good point with adding ellipsis. Can I give couple more points worth to do?

Based on that screenshot: 
<img width="1170" alt="image" src="https://github.com/user-attachments/assets/e670a2bc-a4e8-44a8-9adb-a27cbdc3a95b">


1. You see there is cut "Rules & Rates ..." but when it is selected, then as value of select it's not truncated so user may think it's full value but it's not. 
2. Can you add tooltip for these options to allow user to see whole name? Also please add tooltip if you add truncation to value of select.